html5/js/Utilities.js

```diff
@@ -727,90 +726,46 @@ const LANGUAGE_TO_LAYOUT = {
   zh: "cn",
   af: "za",
   sq: "al",
-  //"ar"	: "ar",
-  //"eu"	: "eu",
-  //"bg"	: "bg",
-  //"be"	: "be",
   ca: "ca",
   "zh-TW": "tw",
   "zh-CN": "cn",
-  //"zh-HK"	: ??
```

Comment:
Please keep those - I don't know what the correct value, but I'm pretty sure it's not `zh`!
If we remove them, this will never be fixed.